treeherder/webapp/api/performance_data.py

```diff
@@ -911,48 +915,42 @@ def list(self, request):
 
         return Response(data=serialized_data)
 
-    def _get_ph_time_ranges(self):
-        return [
-            {'value': 86400, 'text': 'Last day'},
-            {'value': 86400 * 2, 'text': 'Last 2 days'},
-            {'value': 604800, 'text': 'Last 7 days'},
-            {'value': 1209600, 'text': 'Last 14 days'},
-            {'value': 2592000, 'text': 'Last 30 days'},
-            {'value': 5184000, 'text': 'Last 60 days'},
-            {'value': 7776000, 'text': 'Last 90 days'},
-            {'value': 31536000, 'text': 'Last year'},
-        ]
-
-    def _get_push_timestamp(self, base_push, new_push):
+    @staticmethod
+    def _get_push_timestamp(base_push, new_push):
```

Comment:
Retrieving the timestamp for the graph link needs to be updated if there is no base revision.